app/javascript/article-form/components/Header.jsx

```diff
@@ -31,13 +31,17 @@ export const Header = ({
   );
 };
 
+Header.defaultProps = {
+  organisationId: '',
+};
 Header.propTypes = {
   displayModal: PropTypes.func.isRequired,
   onPreview: PropTypes.func.isRequired,
   previewShowing: PropTypes.bool.isRequired,
-  organizations: PropTypes.string.isRequired,
-  organizationId: PropTypes.string.isRequired,
-  onToggle: PropTypes.string.isRequired,
+  organizations: PropTypes.oneOfType([PropTypes.array, PropTypes.string])
+    .isRequired,
```

Comment:
same thing I noted about `organizations` in the `articleForm` component, this goes through the `PageTitle` component which doesn't expect a string

Reply:
I'll check once an update the type